eodatasets3/wagl_md_only.py

```diff
@@ -65,50 +92,57 @@ def package(outdir):
 
                 # note; pathname here is only a relative pathname
                 pathname = GDAL_H5_FMT.format(
-                    filename=str(outdir.joinpath(WAGL_FNAME)),
+                    filename=str(outdir.joinpath(granule.wagl_hdf5)),
                     dataset_pathname=pathname
                 )
 
                 # just for this example, so we insert nbar_blue
                 # otherwise we'll get duplicates if just using blue
                 # something can be done for the other datasets
                 parent = pbasename(ds.parent.name)
+
+                # Get spatial resolution
+                resolution = Path(ds.parent.name).parts[2]
+
+                if resolution == 'RES-GROUP-1':
+                    resolution = 'rg1'
+                else:
+                    resolution = 'rg0'
```

Comment:
In order to support Sentinel-2, you'll need to be generic so that more than two resolution groups are supported.

Something like:

```python
resolution = "rg{}".format(resolution.split("-")[-1])
```